browsers/firefox_android.json

```diff
@@ -434,9 +434,16 @@
         "68": {
           "release_date": "2019-07-09",
           "release_notes": "https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/68",
-          "status": "current",
+          "status": "esr",
```

Comment:
I think should be `retired` too? We don't have any support for 68 at this point.